runtime-modules/content/src/lib.rs

```diff
@@ -5,7 +5,7 @@
 #![allow(clippy::unused_unit)]
 
 #[cfg(test)]
-mod tests;
+pub mod tests;
```

Comment:
I suspect you initially wanted to import the merkle tree utilities from `tests`, but then you copied them to `benchmarks/mod.rs`, so making this module public is no longer needed? Same would apply to `fixtures.rs` etc.
As I suggested before, I think it's best to just move those merkle tree utils to `common` pallet so that we don't need to define them in so many places.